pkg/azurefile/azurefile.go

```diff
@@ -116,6 +116,9 @@ const (
 	fsGroupChangePolicyField          = "fsgroupchangepolicy"
 	ephemeralField                    = "csi.storage.k8s.io/ephemeral"
 	podNamespaceField                 = "csi.storage.k8s.io/pod.namespace"
+	serviceAccountTokenField          = "csi.storage.k8s.io/serviceAccount.tokens"
+	clientIDField                     = "clientID"
```

Comment:
do we support `clientid` field? it could be parsed in `GetAccountInfo`, but it could not be parsed in other places.

Reply:
This filed is not parsed by GetAccountInfo. It must be specified in static PV or ephemeral volume attributes if uses want to use workload identity.